.github/workflows/release.yml

```
@@ -22,7 +22,8 @@ jobs:
         id: get_version
         run: echo ::set-output name=NEW_VERSION::${GITHUB_REF#/refs/tags/v}
       - run: yarn install
-      - run: yarn version --no-git-tag-version --new-version ${{
+      - run:
```

This is needed to fix the interaction with prettier
```suggestion
      - run: |
```

---

Thanks, fixed